CONTRIBUTING.md

```diff
@@ -413,15 +413,28 @@ rule is that they should be as concise as possible.  Specifically:
 * use variable annotations instead of type comments, even for stubs
   that target older versions of Python.
 
-Stub files should only contain information necessary for the type
-checker, and leave out unnecessary detail:
-* for arguments with a default, use `...` instead of the actual
-  default;
-* for arguments that default to `None`, use `Foo | None` explicitly
-  (see below for details);
-* use `float` instead of `int | float`.
+The primary users for stub files are type checkers,
+so stub files should generally only contain information necessary for the type
+checker, and leave out unnecessary detail.
+However, stubs also have other use cases:
+* stub files are often used as a data source for IDEs,
+  which will often use the signature in a stub to provide information
+  on functions or classes in tooltip messages.
+* stubs can serve as useful documentation to human readers,
+  as well as machine-readable sources of data.
```

Comment:
Nit: Capitalize the start of the sentences.

Reply:
All other bullet points in this section currently don't capitalise the start of a bullet point, so (while I agree with you in principle), I'd rather stay consistent with the rest of the section for now